docs/book/v1/features/application.md

```diff
@@ -217,7 +217,7 @@ methods for retrieving them. They include:
   typically a `Mezzio\Emitter\EmitterStack` instance.
 - `getFinalHandler(ResponseInterface $response = null)`: retrieves the final
   handler instance. This is middleware with the signature `function ($request,
-  $response, $error = null)`, and it is invoked when the middleware pipeline
+$response, $error = null)`, and it is invoked when the middleware pipeline
```

Comment:
This is ambiguous change. I am not sure it is entirely correct.
Can you please reformat it to put code span on a single line?